flang/lib/Semantics/resolve-directives.cpp

```diff
@@ -1674,7 +1674,6 @@ void OmpAttributeVisitor::ResolveSeqLoopIndexInParallelOrTaskConstruct(
   // parallel or task
   if (auto *symbol{ResolveOmp(iv, Symbol::Flag::OmpPrivate, targetIt->scope)}) {
     targetIt++;
-    symbol->set(Symbol::Flag::OmpPreDetermined);
```

Comment:
I would prefer retaining this. Can we skip if it is predetermined and a parallel region in the DataSharingprocessor?

Reply:
Will do ....